helm/loki/values.yaml

```diff
@@ -90,7 +90,18 @@ loki:
   enabled: true
 
   gateway:
+    extraContainers:
+      - name: dnsmasq
+        image: "janeczku/go-dnsmasq:release-1.0.7"
```

Comment:
I think we will have to retag that image but otherwise looks good

Reply:
=> image retag and repo `go-dnsmasq` created on `quay` and `aliyun`